api/v1alpha1/check_types.go

```diff
@@ -90,6 +90,11 @@ func (c *Check) GetResources() corev1.ResourceRequirements {
 	return c.Spec.Resources
 }
 
+// GetSecurityContext returns the security context
+func (c *Check) GetSecurityContext() *corev1.PodSecurityContext {
```

Comment:
...and all the other types that I'm too lazy to mark ;)
```suggestion
// GetSecurityContext returns the pod security context
func (c *Check) GetSecurityContext() *corev1.PodSecurityContext {
```